controllers/blogsController.js

```diff
@@ -0,0 +1,85 @@
+import Blog from "../model/Blog.js";
+import cloudinaryUpload from "../config/cloudinaryUpload.js";
+
+export const getAllBlogs = async (req, res) => {
+  const blogs = await Blog.find();
+  if (!blogs) return res.status(204).json({ message: "No blogs found" });
+  res.json(blogs);
+};
+
+export const createNewBlog = async (req, res) => {
+  if (
+    !req?.body?.title ||
+    !req?.body?.body ||
+    !req?.body?.date ||
```

Comment:
you have middleware that validates all request from body there is no need of this validation

Reply:
Oh yes, I forgot to remove that one